tensorflow/core/kernels/data/name_utils.h

```diff
@@ -0,0 +1,59 @@
+/* Copyright 2019 The TensorFlow Authors. All Rights Reserved.
```

Comment:
For all the public methods in this API, provide an example in the documentation (given inputs "foo", method returns "bar").

Reply:
Done